shenyu-bootstrap/src/main/resources/application.yml

```diff
@@ -253,6 +253,7 @@ shenyu:
 #      namespace: 1c10d748-af86-43b9-8265-75f487d20c6c
 #      username:
 #      password:
+#      context-path:
```

Comment:
default context path is nacos?

Reply:
yes,the same as shenyu-admin/src/main/resources/application.yml